k8s.gcr.io/images/k8s-staging-krm-functions/images.yaml

```diff
@@ -0,0 +1 @@
+# No images yet
```

Comment:
What is the process to update this file? Is it manual?

Reply:
Yes, it is manual. We can create and store the images in the staging GCP project that I'm trying to create here. Adding the image SHAs here will publish the images as official kubernetes images.

Here is a sample PR from kustomize: https://github.com/kubernetes/k8s.io/pull/3387. Those are SHAs of images that are currently stored in kustomize's staging repo. After that PR goes in, the images will be published here: https://pantheon.corp.google.com/gcr/images/k8s-artifacts-prod.

I am expecting we can do the same process for the function images.